docs/design-proposals/multi-tenancy/multi-tenancy-api-resource-model.md

```diff
@@ -168,14 +169,22 @@ Here are some examples:
  2. **Manageability**. A cluster operator (he should be user under the system tenant) can introduce a new CRD, where he can choose to:
   * make it usable only to the system tenant.
   * or, make it usable only to a spefic set of to regular tenants and the system tenants, 
-  * or, usable to all the regular tenants. 
+  * or, usable to all the tenants.
   * Additionally, the applicable range of a CRD can be changed by the cluster operator at any time in a simple and quick manner. That will be useful to CRD test and rolling update.  
 
-  (*** What the managebility should look like is not finalized as we are still working on it. )
+
+  
+  When a system CRD is usable to regular tenants, this sharing can be:
+  * **forced**. The system CRD will be usable by all the tenants. It overrides the local CRDs with the same name in non-system tenant spaces. Once a forced system CRD is deployed, a regular tenant's attempt to create a CRD with the same name will be rejected. 
+
+    Cluster admins are suggested to use it only for those resource types mandatory to every tenant.  
+  * **optional**. The regular tenant is allowed to override such system CRDs by defining a local CRD.
```

Comment:
so this "optional"  sounds like "tenant first"? If that is supported, what would be the results of get such CR --tenant=all?

Reply:
When list CR --tennat=all, we should only list the CR from the tenants that uses this system CRD. The tenants, who chooses to use their own CRD, actually are using a different type of resource.

As mentioned in the doc, only "forced" CRD's behavior is implemented so far. So the behavior for those optional CRDs are subject to change per real use scenarios.